src/app/integrations/edit-page/flow-view/flow-view-step.component.html

```diff
@@ -30,8 +30,11 @@ <h4 class="modal-title pull-left">Confirm Delete</h4>
 
   <div class="col-md-3 progress-line">
     <div class="icon">
-      <p [class]="'round ' + getActiveClass()">
-        <i [class]="getIconClass()"></i>
+      <p [class]="'round ' + getSubMenuActiveClass()"
+         [class.add-step-or-connection]="step.stepKind !== ('endpoint' || 'log')">
```

Comment:
the `log` step doesn't need a special icon, only `endpoint` have an icon now, so maybe this check isn't needed?